Yoast/wordpress-seo
# Updated most lodash uses

## Summary

This PR can be summarized in the following changelog entry:

* Makes sure no lodash-version prior to `4.17.11` is used.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

For developers:
* Run `yarn why lodash` and make sure no lodash version prior to `4.17.11` is used.

## UI changes

## Quality assurance


Fixes #12470
